src/handlers/testing/align/util/hono-app.ts

```diff
@@ -0,0 +1,90 @@
+import { zValidator } from '@hono/zod-validator';
+import { Hono, type Context } from 'hono';
+import { SafeParseReturnType, z } from 'zod';
+import { Language } from './models';
+import { SessionManager } from './session-manager';
+
+const handleValidationResult = (
+  result: SafeParseReturnType<unknown, unknown>,
+  c: Context,
+) => {
+  if (!result.success) {
+    return c.json('Bad Request', 400);
+  }
+};
+
+/**
+ * Server that receives requests from the SDKs
+ */
+export function createHonoApp(sessionManager: SessionManager): Hono {
```

Comment:
we can probably share the same server across both `exec` and `align` and pass callbacks within the createHonoApp function instead